src/apis/platformvm/importtx.ts

```diff
@@ -88,7 +88,7 @@ export class ImportTx extends BaseTx {
    */
   toBuffer():Buffer {
     if(typeof this.sourceChain === "undefined") {
-      throw new Error("ImportTx.toBuffer -- this.sourceChain is undefined");
+      throw new Error(`ImportTx.toBuffer -- ${this.sourceChain} is undefined`);
```

Comment:
Would an error message of `ImportTx.toBuffer -- given sourceChain ${this.sourceChain} is undefined` be more helpful, or is that redundant?

Reply:
Sure I think that makes sense. Updated.